styles/styles.css

```
@@ -0,0 +1,156 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format ("woff2")
```

```suggestion
  src: url("../fonts/ptsans-400.woff2") format("woff2")
```
Лишний пробел и в результате браузер шрифт не подключает.